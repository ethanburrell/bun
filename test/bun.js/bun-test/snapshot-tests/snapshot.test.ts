import {
    describe,
    expect,
    it,
  } from "bun:test";

describe("test generic snapshot tests", () => {
    describe("inner",() => {
        it("test single snapshot test", () => {
            expect("hello").toMatchSnapshot()
        })
    })
    it("test two snapshot tests", () => {
        expect("hello").toMatchSnapshot()
        expect("hi there").toMatchSnapshot()
    })
})
