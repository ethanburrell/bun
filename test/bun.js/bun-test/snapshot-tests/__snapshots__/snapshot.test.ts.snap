// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test generic snapshot tests test two snapshot tests 1`] = `hello`;

exports[`test generic snapshot tests test two snapshot tests 2`] = `hi there`;

exports[`test generic snapshot tests inner test single snapshot test 1`] = `hello`;
